core/common/src/main/java/alluxio/wire/ThriftUtils.java

```diff
@@ -122,6 +124,17 @@ public static WorkerNetAddress fromThrift(alluxio.thrift.WorkerNetAddress worker
   }
 
   /**
+   * Converts thrift type to wire type.
+   *
+   * @param tTtlAction {@link TTtlAction}
+   * @return {@link TtlAction} equivalent
+   */
+  public static TtlAction fromThrift(TTtlAction tTtlAction) {
+
+    return TtlAction.fromTTtlAction(tTtlAction);
```

Comment:
similar to other methods in this class this should be

```
return new TtlAction(tTtlAction)
```

In other words, instead of `fromTTlAction` factory method the TtlAction class should have a constructor that takes a thrift object. This is for consistency with how other classes in this package work.